package.json

```diff
@@ -68,8 +68,11 @@
     "@google-cloud/nodejs-repo-tools": "^2.3.0",
     "@types/mocha": "^5.2.3",
     "@types/node": "^10.3.5",
+    "chai": "^4.1.2",
```

Comment:
Are you using chai and chai-as-promised? This seems a little weird.

Reply:
This PR is meant to make https://github.com/googleapis/nodejs-firestore/pull/242 smaller. I want to start using `chair` in my tests. 

See https://github.com/googleapis/nodejs-firestore/pull/242/files#diff-f626ff6b37edf53ad3138d2e458d5f44R41